pkg/secureframe/graphql.go

```diff
@@ -308,13 +345,13 @@ func query(ctx context.Context, token string, in interface{}, out interface{}) e
 		return fmt.Errorf("read: %w", err)
 	}
 
-	// log.Printf("response: %s", rb)
+	log.Printf("response: %s", rb)
```

Comment:
Can these be commented out now that the changes work?

Reply:
Yes, but I'd prefer to leave them in as any breakage almost certainly means uncommenting these again.

Reply:
Yea, good point 🥲